frontend/src/routes/Governance/common/util.tsx

```diff
@@ -467,23 +468,45 @@ export function getPolicyRemediation(policy: Policy | undefined) {
   if (!policy) {
     return ''
   }
-  if (policy.spec?.remediationAction) {
-    return policy.spec.remediationAction
-  }
   const templates = policy.spec['policy-templates']
-  let remediationAggregation = '-'
+  let remediationAggregation = ''
+
+  const remediationSet = new Set()
+
   templates?.forEach((template: PolicyTemplate) => {
     const templateRemediation = template.objectDefinition.spec.remediationAction
-    if (remediationAggregation === 'inform/enforce') {
-      return
-    } else if (remediationAggregation !== '-' && remediationAggregation !== templateRemediation) {
-      remediationAggregation = 'inform/enforce'
-      return
-    } else if (remediationAggregation !== templateRemediation && templateRemediation) {
-      remediationAggregation = templateRemediation
+    if (remediationSet.size === 3) {
       return
+    } else {
+      remediationSet.add(templateRemediation)
     }
   })
+
+  if (remediationSet.has('inform')) {
+    remediationAggregation += 'inform '
+  }
+  if (remediationSet.has('enforce')) {
+    remediationAggregation += 'enforce '
+  }
+  if (remediationSet.has('informOnly')) {
+    remediationAggregation += 'informOnly'
+  }
+
+  remediationAggregation = remediationAggregation.trim().replaceAll(' ', '/')
+  if (remediationAggregation === '') {
+    remediationAggregation = '-'
+  }
+  if (policy.spec?.remediationAction) {
+    if (remediationAggregation.includes('informOnly')) {
+      if (remediationSet.size > 1) {
+        return policy.spec.remediationAction.concat('/informOnly')
+      } else {
+        return 'informOnly'
+      }
+    } else {
+      return policy.spec.remediationAction
+    }
+  }
   return remediationAggregation
```

Comment:
It could create "enforce/inform" or "informOnly/enforce" so I think we need to sort it

Reply:
In lines 485-493, the condition statements should always make it so that the order is always `inform` -> `enforce` -> `informOnly`. I don't think `enforce` will ever precede `inform`, and similar with the other examples. I'm constructing a string from the set, and not using the set directly, so I don't think we need to sort it.